source/js/header.js

```diff
@@ -162,14 +162,13 @@
         }
       }
       // Check for browser locale
-      else if(navigator.language.split('-')[1] === 'US' || navigator.userLanguage.split('-')[1] === 'US'){
+      else if(navigator.language && navigator.language.split('-').length > 1 && navigator.language.split('-')[1] === 'US') {
```

Comment:
You might have to change this a bit cause navigator.language is for chrome and safari and navigator.user.language is for IE so your logic would not work for IE.

Reply:
i added a section for IE below. This would likely have failed in IE before since not checking if navigator.language is undefined would throw an error on split